core/vibes/soul/primitives/accordions/index.tsx

```diff
@@ -0,0 +1,99 @@
+'use client';
+
+import * as AccordionsPrimitive from '@radix-ui/react-accordion';
+import { clsx } from 'clsx';
+import React from 'react';
+
+function Accordion({
+  children,
+  title,
+  colorScheme = 'light',
+  ...rest
+}: React.ComponentPropsWithoutRef<typeof AccordionsPrimitive.Item> & {
+  colorScheme?: 'light' | 'dark';
+}) {
+  return (
+    <AccordionsPrimitive.Item {...rest}>
+      <AccordionsPrimitive.Header>
+        <AccordionsPrimitive.Trigger asChild>
+          <div className="group cursor-pointer items-start gap-8 py-3 last:flex @md:py-4">
+            <div
+              className={clsx(
+                'flex-1 select-none font-[family-name:var(--accordion-title-font-family)] text-sm uppercase transition-colors duration-300 ease-out',
+                {
+                  light:
+                    'text-[var(--accordion-light-title-text)] group-hover:text-[var(--accordion-light-title-text-hover)]',
+                  dark: 'text-[var(--accordion-dark-title-text)] group-hover:text-[var(--accordion-dark-title-text-hover)]',
+                }[colorScheme],
+              )}
+            >
+              {title}
+            </div>
+            <AnimatedChevron
+              className={clsx(
+                {
+                  light:
+                    'stroke-[var(--accordion-light-title-icon)] group-hover:stroke-[var(--accordion-light-title-icon-hover)]',
```

Comment:
Seems like this variable is not set, resulting in icons being invisible

![CleanShot 2024-12-15 at 23 09 43@2x](https://github.com/user-attachments/assets/cff80a73-9301-42e9-908f-9f6407db69a4)

Reply:
Ok, just pushed up the missing variables